cmd/osbuild-composer/composer.go

```diff
@@ -107,6 +108,9 @@ func (c *Composer) InitWeldr(repoPaths []string, weldrListener, localWorkerListe
 
 	// TODO: refactor to be more generic
 	name := hostDistro.Name()
+	if name == "rhel-84" {
+		name = "rhel-8"
+	}
```

Comment:
Will we eventually split up the rhel-8 distro into it's minor releases? If not should I make this more generic now?

Reply:
I think that would make sense. There are no immediate plans to split into the minor releases, that would anyway require more work as some customers would be on the generic RHEL8, but others would opt into a specific minor version.